css/constitution.md

```diff
@@ -30,6 +30,332 @@ Society Membership shall consist of:
+
+#### Section 3 - Meeting to Remove an Executive or Council Member
+
+An **Executive** meeting shall be called to determine whether a **member**
+shall continue to retain their position.
+
+-   At least _one-week_ notice shall be given before the meeting. The
+    **member** in question will be given an opportunity to address any issues
+    and concerns raised, answer any questions as well as provide reasons why
+    they should continue to hold their position.
+-   Voting shall take place by secret ballot.
+-   Quorum for the meeting shall be all of the **executives**, not including the accused.
+-   If quorum is not met, or in the event that a **two-thirds vote** is
+    not obtained, the individual shall retain their position.
+-   No minutes shall be published for this meeting, beyond the immediate
+    result.
+
+#### Section 4 - Constitutional Amendments
+
+The constitution can be amended at a regular meeting, or a general
+meeting, with a three-fourths majority vote.
+
+Any amendments must be announced one week prior to either meeting.
+
+#### Section 5 - Interpretation of Bylaws
+
+These bylaws are to be interpreted by the Executive.
+
+#### Section 6 - Conflicts with UWSA Policy
+
+Nothing in this instrument shall be read to contradict UWSA rules
+regarding clubs or societies. In the event where a contradiction shall
+occur, the rules of the UWSA shall have precedent.
+
+#### Section 7 - Parliamentary Authority
+
+The rules contained in the current edition of Robert's Rules of Order
+Newly Revised shall govern CSS in all cases to which they are applicable
+and in which they are not inconsistent with these bylaws and any special
+rules of order CSS may adopt.
+
+#### Section 8 - Resignations
+
+An executive member may resign from their position by submitting a
+written notice to the Executive at least two weeks in advance. The
+President shall be responsible for filling the vacant position, by
+appointment or election, as the President deems necessary.
+
+# In the case of the resignation of the President, the position shall be
+
+---
+
+id: constitution
+title: Constitution
+sidebar_label: Constitution
+slug: /
+
+---
+
+### Article 1.
+
+#### Section 1 - Name
+
+The name of this organization shall be the Computer Science Society.
+
+#### Section 2 - Purpose
+
+The objective of the society shall be to promote an interest in Computer
+Science and to enhance the lives of computer science students on campus.
+
+#### Section 3 - Membership
+
+Society Membership shall consist of:
+
+1. All students who are currently enrolled on a full time basis at the
+   University of Windsor majoring in a program offered by the department of
+   Computer Science.
+2. Any student at the University of Windsor, who is not otherwise
+   covered by part a, may purchase a membership for **$20 per semester**.
+
+### Article 2.
+
+#### Section 1 - Positions
+
```

Comment:
Looks like everything above this is a duplicate of everything below. Possible residue from a merge conflict.

Reply:
nope, this is the exact file I copied from your update on the master branch